devo/api/client.py

```diff
@@ -605,8 +626,8 @@ def _get_sign(self, data, tstamp):
         """
         if not self.auth.get("key", False) \
                 or not self.auth.get("secret", False):
-            raise DevoClientException(_format_error("You need a API Key and "
-                                                    "API secret to make this"))
+            raise DevoClientException(("You need a API Key and "
+                                                    "API secret to make this"),400)
```

Comment:
If we don't have address, we cannot create an HTTP request and no HTTP status code can be returned. It will create confussion in developer/use as no HTTP service can have returned any status code. Do not create HTTP status codes. Just returns the one returned by Serrea.